File: benchmark/results/java-invocation-inline/run-004/b-baseline/output/com/metaengine/demo/ordering/services/OrderValidator.java
package com.metaengine.demo.ordering.services;

import java.util.List;
import com.metaengine.demo.ordering.aggregates.Order;

/** OrderValidator domain service. */
public class OrderValidator {
    /** Validate and create an Order. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Order by id; returns null if not found. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to limit Orders. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Order by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-004/b-baseline/output/metaengine_demo/ordering/services/order_validator.py
"""OrderValidator service module."""
from typing import Optional

from metaengine_demo.ordering.aggregates.order import Order


class OrderValidator:
    """OrderValidator service."""

    def create(self, input: Optional[Order]) -> Order:
        """Create an Order."""
        raise NotImplementedError("not implemented")

    def find_by_id(self, id: str) -> Optional[Order]:
        """Find an Order by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: int) -> list[Order]:
        """List Orders."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete an Order by id."""
        raise NotImplementedError("not implemented")
